wiki/Modding/en.md

```diff
@@ -1,262 +1,42 @@
-
-### Background, Skin, and Combo Colours
-
-All beatmaps must have a background image.
-The skin you supply should not blend with the background image or greatly hinder (or chellenge) game play.
-For example, complicated hit circle design, huge hit bursts, etc.
+**Modding** is the act of providing feedback on another user's [beatmap](/wiki/Beatmaps). A user who is modding is referred to as a *modder*, and their collective posts on a beatmap's discussion are referred to as a *mod*.
 
-There are also many [skinning](/wiki/skinning) rules that should be followed.
-Similar to skinning, combo colours should not blend in with the background either.
-These combo colours should also be very distinct and clear.
-The mapper should not have two almost identical combo colours unless they are separated by a very contrasting colour in between.
+Mods are mainly used to improve beatmaps aiming for Ranked status. Because it is a detailed and analytical process, modding beatmaps is also a popular way for users improve their understanding of beatmapping.
 
-### Storyboarding
+## Finding beatmaps to mod
 
-*Main page: [Storyboarding](/wiki/Storyboarding)*
+[Modding Queues](https://osu.ppy.sh/community/forums/60) are the most actively utilized platform for finding beatmaps to mod and requesting mods. A user can create their own forum thread, referred to as a *queue*, for other users to post beatmaps that they want to be modded.
 
-If you spot anything weird while the storyboard is playing (e.g. bad rendering of pictures, emptiness, etc.), it is a good idea to tell the [storyboarder](/wiki/storyboarder) so that they can fix it.
-You can check out the storyboard by pressing `F2` or clicking the `Design` tab at the top of the screen in editor.
+Each queue's creator dictates the queue's rules, controlling which beatmaps are viable for receiving mods. Queues are usually open for requests in two ways:
 
-The dimensions of the elements should not exceed the default size.
+- **Normal mod (NM):** A mod that will be provided without work from the requesting beatmapper.
+- **Mod for mod (M4M):** A trade that requires each user to mod the other's beatmap.
 
-### Preview Point
+The `#modreqs` chat channel is populated by beatmappers posting maps they want modded. Most beatmaps posted do not receive mods.
 
-All maps must have a preview point.
+Well-known modders often note whether or not they are accepting mod requests on their user profiles.
 
-A preview point will be denoted by a yellow tick mark on the [timeline](/wiki/timeline).
-You can also check if a map has a preview point by going into [AiMod](/wiki/AiMod) and checking under the `Style` tab.
+Searching the pending beatmaps listing for beatmaps to mod is not recommended because most submissions are not aiming for Ranked status, so their creators will potentially not respond to mods.
 
-The preview point must be the same in all difficulties.
+## Modding process
```

Comment:
something like "How to mod" is probably better cuz u speak to the reader directly in this part